static/app/app/list/views.js

```diff
@@ -27,12 +27,28 @@ var LunchMoveView = Marionette.ItemView.extend({
         'editMove': '.own-move',
         'addMove': '[data-ui="addMove"]'
     },
+    initialize: function(params) {
+        this.recentlySaved = params.recentlySaved;
+    },
     addMove: function(e){
         e.preventDefault();
         channel.request('entities:move').set('spot', this.model.id);
         channel.trigger('edit');
         return false;
     },
+    onShow: function() {
+        if (this.recentlySaved) {
+           this.recentSaveAlert();
+        }
+    },
+    recentSaveAlert: function() {
+        debugger
+        var $moveBox = this.ui.editMove;
+        $moveBox.addClass('background-flash');
+        setTimeout(function(){
+            $moveBox.removeClass('background-flash');
+        }.bind(this), 600)
```

Comment:
you're binding the context, so storing this.ui.editMove in a local variable is unnecessary. You can just refer to it as `this.ui.editMove` in the callback too.